libs/cart/src/drivers/interfaces/cart-item-service.interface.ts

```diff
@@ -0,0 +1,14 @@
+import { DaffCartItem } from '../../models/cart-item';
+import { Observable } from 'rxjs';
+import { InjectionToken } from '@angular/core';
+
+export interface DaffCartItemServiceInterface<T extends DaffCartItem> {
+	list(): Observable<T[]>;
+	get(id: T['item_id']): Observable<T>;
+	update(itemId: T['item_id'], changes: any): Observable<T>;
```

Comment:
What would `changes` be here? Just the cart item itself? This should be typed if possible.

Reply:
I would do what you did with the cart payment service and do `changes: T`.

Reply:
@griest024 I've typed this a Partial, let me know what you think.

Reply:
lgtm